project/assets/login.less

```diff
@@ -0,0 +1,99 @@
+body{
+  background-color: rgb(82, 86, 89);
+  margin-top: 50px;
+  margin-left: 0;
+}
+
+.login-form {
+  opacity: 0.8;
+
+  .login-form-background-image {
+    width: 1897px;
```

Comment:
Starting to see a pattern with this `1897px` width. Not sure what its for tho...